packages/object-lists/src/db/schema.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  index,
+  timestamp,
+  mysqlTable,
+  serial,
+  varchar,
+  text,
+  unique,
+  int,
+} from 'drizzle-orm/mysql-core';
+import {sql, relations} from 'drizzle-orm';
+
+export const objectLists = mysqlTable(
+  'object_list',
+  {
+    id: serial('id').primaryKey(),
+    name: varchar('name', {length: 256}),
+    description: text('description'),
+    communityId: varchar('community_id', {length: 256}),
+    createdAt: timestamp('created_at')
+      .default(sql`CURRENT_TIMESTAMP`)
+      .notNull(),
+    createdBy: varchar('created_by', {length: 256}).notNull(),
```

Comment:
To be sure: is the `length` in sync with the Clerk definitions?

Reply:
Can we ask Clerk about this? If we're unsure then even a high max of 256 could be insufficient. And if Clerk always uses a length of 32 (sounds like a UUID?) then 256 is way too high.

Reply:
I found the answer somewhere in Discord:

![image](https://github.com/colonial-heritage/colonial-collections/assets/1481602/48fb01e5-4630-454b-a735-43aaf4f0384b)

I will follow the advice and make it 50.